modules/bag/CMakeLists.txt

```diff
@@ -26,7 +27,13 @@ set(SOURCES
 # library target
 define_module_library(
   NAME bag
-  PUBLIC_LINK_LIBS absl::base hephaestus::containers hephaestus::ipc hephaestus::telemetry mcap
+  PUBLIC_LINK_LIBS
+    absl::base  
+    hephaestus::containers
+    hephaestus::ipc
+    hephaestus::telemetry
+    mcap
+    zstd::libzstd
```

Comment:
I am happy to add `zstd` support but it will have to be added also to `externals/CMakeLists.txt` with `add_cmake_dependency` and to Bazel.

For bazel you can just add `bazel_dep(name = "zstd", version = "1.5.6")` into `MODULE.bazel` and then link in  `mcap.BUILD`  
 
Let me know if you need help with that.

Reply:
I honestly didn't realize that zstd isn't present anywhere before upstream, because our cmake wrapper had it as a dependency already in there. It's a an optional dependency of mcap IIRC and there are already todo's in bazel to add it, exactly as you suggested. will check.

Reply:
here I do not use the official Bazel wrap of mcap as it is not great, so please have a look at `mcap.BUILD` you just need to add a dependency to `zstd` and remove `MCAP_COMPRESSION_NO_ZSTD`